cfme/common/vm_views.py

```diff
@@ -317,6 +317,7 @@ class ProvisionView(BaseLoggedInPage):
     title = Text('#explorer_title_text')
     breadcrumb = BreadCrumb()
     image_table = Table('//div[@id="pre_prov_div"]//table')
```

Comment:
@lina-nikiforova can you please confirm that are you able to cover this table with `BaseEntitiesView`. Just simple though comes in my mind. If yes then it will solve your problem.